src/test/resources/pages/instructorStudentRecords.html

```diff
@@ -112,7 +112,7 @@
                   <td>
                     Ben (
                     <i>
-                      male
+                      MALE
```

Comment:
Here, I think we need to display gender in lowercase as previously. What you need to do to pass old version of this test without it editing. You need to add the following method to teammates.ui.template.StudentProfile to display gender type in the old way:
```java
    public String getDisplayedGenderType() {
        return gender == null || gender == GenderType.OTHER
                ? Const.STUDENT_PROFILE_FIELD_NOT_FILLED
                : gender.name().toLowerCase();
    }
```
After that, you need to check WEB-INF/tags/instructor/courseStudentDetails/studentProfile.tag. At line 26 will see: 
```jsp
${student.gender == other ? noneForGender : fn:escapeXml(student.gender)}
``` 
We don't need `fn:escapeXml` anymore because enum don't contains characters that could be interpreted as XML markup we can safely replace expression above with getDisplayedGenderType() method you added previously. 
To call that method you can use `${student.displayedGenderType}` or `<%=  student.getDisplayedGenderType() %>`.
After that you can remove constants in lines 8-9, since they don't used anymore and rerun teammates.test.cases.browsertests.InstructorStudentRecordsPageUiTest#testAll to ensure that gender type is displayed in old way:
```jsp
<c:set var="noneForGender"><span class="text-muted"><%= Const.STUDENT_PROFILE_FIELD_NOT_FILLED %></span></c:set>
<c:set var="other"><%= GenderType.OTHER %></c:set>
```
Sorry, that I didn't noticed this during previous review.